src/hist/basehist.py

```diff
@@ -210,6 +211,45 @@ def fill(
         total_data = tuple(args) + tuple(data)  # Python 2 can't unpack twice
```

Comment:
Totally unrelated, but we should unpack twice, hist has never supported Python 2... :)